src/engines/kotorbase/area.cpp

```diff
@@ -241,13 +241,20 @@ void Area::show() {
 
 	GfxMan.lockFrame();
 
+	std::set<Common::UString> visibleRooms(getRoomsVisibleByPartyLeader());
+
 	// Show rooms
-	for (RoomList::iterator r = _rooms.begin(); r != _rooms.end(); ++r)
-		(*r)->show();
+	for (auto r : _rooms) {
```

Comment:
should be auto &, and likewise below

Reply:
Why though? It would be a reference to a pointer. Makes no sense to me :P